libs/wire-api/src/Wire/API/Conversation.hs

```diff
@@ -232,6 +235,19 @@ instance ToSchema (Versioned 'V2 ConversationMetadata) where
           "ConversationMetadata"
           (conversationMetadataObjectSchema accessRolesSchemaV2)
 
+-- | Client list for internal API.
+data ClientList = ClientList {clClients :: [ClientId]}
```

Comment:
I want to move this somewhere, but unsure what the best place for it is. Perhaps new module with this + the handling function for the internal API call. Thoughts?